web/src/components/footer/subcomponents/brand/brand.js

```diff
@@ -0,0 +1,23 @@
+import { Box, Typography } from "@mui/material";
+import prideLogo from "../../../../assets/images/color_logo_512.png";
+import brandStyles from "./brand.styles";
+import messages from "../../../../common/messages";
```

Comment:
`import messages from "@/common/messages";`
https://stackoverflow.com/questions/42711175/what-does-the-symbol-do-in-javascript-imports